spot_driver/spot_driver/spot_ros2.py

```diff
@@ -1290,7 +1291,7 @@ def main(args=None):
                 time.sleep(0.5)
             print('Found estop!', flush=True)
 
-        node.create_timer(0.1, spot_ros.step, callback_group=spot_ros.group)
+        node.create_timer(0.01, spot_ros.step, callback_group=spot_ros.group)
```

Comment:
Can we make this a parameter that defaults to `0.1`?  Saves us a bunch of testing before letting the PR go through.